_source/handbook/02_drive.md

```diff
@@ -75,8 +77,6 @@ Restoration visits cannot be canceled and do not fire `turbo:before-visit`. Turb
 
 ## Custom Rendering
 
-Turbo Drive's default rendering replaces the contents of the requesting document's `<body>` element with the contents of the response document's `<body>` element.
```

Comment:
I think this first paragraph was ok, since it introduces the behavior that you can change with a customer renderer. Do you find it misleading?

Reply:
I removed it because the reader already knows that. At the beginning of the guide the reader learns about the default rendering, and here they are going to learn about ways to customize it. If you look at [the guide](https://turbo.hotwired.dev/handbook/drive), the distance is kind of short.

However, glad to restore it if you prefer. If we do it, I think it should be reworded because this description is partial and could be misleading. Repeating the same does not make sense to me. Maybe we could remind the user default rendering was explained above or something. What do you think?

Reply:
I see. The partial description doesn't look misleading to me, but maybe it's because I'm familiar with Turbo. I agree it's redundant in any case.

Reply:
Could be. Let me elaborate :).

I believe it is misleading in the sense that if the docs say Turbo Drive substitutes the body, then it follows it does not update meta tags or the page title or does anything else.

Therefore, users like me may wonder if they have to change the title manually. Or why is the csfr token changing at all. That contradicts the docs!

You may say, "oh yes, Turbo Drive does more stuff". Precisely, that is what the docs have to explain to users.